boards/arm/mps2_an385/mps2_an385.dts

```diff
@@ -15,6 +15,7 @@
 		led1 = &led_1;
 		sw0 = &user_button_0;
 		sw1 = &user_button_1;
+		watchdog-0 = &wdog0;
```

Comment:
Would be nice to have consistency between zephyr aliases, so 'watchdog0'

Reply:
Yeah, I wasn't sure about this one.  There are aliases for some SoCs that are of the form `<FOO>-<n>`.

Reply:
Look at dts/arm/nxp/nxp_k8x.dtsi

Reply:
Indeed, but these are not zephyr aliases (as the ones that can be used in samples).
I don't know what they're needed for, but looks like vendor specifics, and possibly reuse from Linux (as I've seen such there as well).

Reply:
Changed to `watchdog0`

Reply:
Now I'm torn, we seem to have a mix of with and without the `-` used for samples & tests aliases.  See PR #20334